Library/Homebrew/compat/hbc/cli/update.rb

```diff
@@ -0,0 +1,17 @@
+module Hbc
```

Comment:
> this one is included as-is regardless of a `--no-compat`

No, it isn't. Everything under `compat/hbc` is not included when `--no-compat` is specified.